services/mock-service/src/fsp-integration/intersolve-visa/intersolve-visa.mock.service.ts

```diff
@@ -550,4 +550,62 @@ export class IntersolveVisaMockService {
       status: HttpStatus.OK,
     };
   }
+
+  public linkToken(
+    _childTokenCode: string,
+    parentTokenCode: string,
+  ): IntersolveVisaMockResponseDto {
+    if (parentTokenCode.includes('mock-fail-link-token')) {
+      return {
+        status: HttpStatus.NOT_FOUND,
+        statusText: 'Not Found',
+        data: {
+          success: false,
+          errors: [
+            {
+              code: 'NOT_FOUND',
+              description: 'We mocked that linking the token failed',
+            },
+          ],
+        },
+      };
+    }
+    return {
+      status: HttpStatus.OK,
+      statusText: 'OK',
+      data: {
+        success: true,
+      },
+      errors: [],
+    };
+  }
+
+  public transfer(parentTokenCode: string): IntersolveVisaMockResponseDto {
```

Comment:
Consider renaming the parameter into fromTokenCode, since there is no requirement this is a parent token. The funding token does not have children.